### DISCUSSION TITLE
How to remap d to "_d in Vim mode

### DISCUSSION THREAD

**Discussion by idr4n** [zed-industries/zed#13659] [Feature Requests]
Hi,

I'm trying to remap `d` to `"_d` but haven't figured it out yet.

Based on #12895, vim registers were added but it seems like they work only when `"use_system_clipboard": "always",`.  I have tried this but didn't work:

```json
  {
    "context": "Editor && vim_mode == normal",
    "bindings": {
      "d d": ["workspace::SendKeystrokes", "\" _ d d"],
    }
}
```

However, this works `"m m": ["workspace::SendKeystrokes", "\" + d d"]` if `"use_system_clipboard": "on_yank",`, meaning there is some support for mapping registers in that way.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
